packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.css

```diff
@@ -1,7 +1,15 @@
+:host {
+  --slds-c-toast-sizing-min-width: 100%;
```

Comment:
Why is this necessary?

Reply:
I documented that in the PR description, it is to fix the width issue of the notifications